tools/pkgutils.py

```diff
@@ -8,99 +9,101 @@
 
 # Figure out what type of package to build based on platform info
 #
-# TODO: Windows does MSI?
+# Below is the task to_do but did not add annotation as it is giving warning in pylint code analysis
+# Windows does MSI?
 
-deb = ['debian', 'ubuntu']
-rpm = ['redhat', 'fedora', 'suse', 'opensuse', 'centos']
+DEB = ['debian', 'ubuntu']
+RPM = ['redhat', 'fedora', 'suse', 'opensuse', 'centos']
 
-dist = platform.dist()[0].lower()
+DIST = platform.dist()[0].lower()
 
 def read_config():
+    """ Module to read config. """
```

Comment:
This is not a module. It's a function. The function name, `read_config` clearly says it's reading a config. What config? Why? Who would call this?